app/crowdSourceData/developer.json

```diff
@@ -94,18 +137,41 @@
     "github_url": "https://github.com/prajwalstha123",
     "linkedin_url": "https://www.linkedin.com/in/prajwalstha"
   },
- {
-  "name": "Kusum Gole",
-  "email": "kusumgole67@gmail.com",
-  "designation" : "Quality Assurance Lead",
-  "experience" : 5,
-  "skills" : ["Manual testing","Test cases","Test Strategy","Automation Testing"],
-  "current_company" : "Outside",
-  "is_intern" : false,
-  "actively_job_searching": false,
-  "address" : "Kathmandu, Nepal",
-  "state" : "3",
-  "github_url" : "https://github.com/Ksmgole" ,
-  "linkedin_url" : "https://www.linkedin.com/in/kusum-gole-244817102/"
-}
-]
+  {
+    "name": "Kusum Gole",
+    "email": "kusumgole67@gmail.com",
+    "designation": "Quality Assurance Lead",
+    "experience": 5,
+    "skills": [
+      "Manual testing",
+      "Test cases",
+      "Test Strategy",
+      "Automation Testing"
+    ],
+    "current_company": "Outside",
+    "is_intern": false,
+    "actively_job_searching": false,
+    "address": "Kathmandu, Nepal",
+    "state": "3",
+    "github_url": "https://github.com/Ksmgole",
+    "linkedin_url": "https://www.linkedin.com/in/kusum-gole-244817102/"
+  },
+  {
+    "name": "Jenish Shrestha",
+    "email": "shrestha.jenish1@gmail.com",
+    "designation": "Software Engineer",
```

Comment:
@jenish1231 please don't format other json data too. Can you revert and send it once again ?